### PR TITLE
ISCT-137_164 button, icon button, loader components v4

### PR DESCRIPTION
**Pull request by AndyKIron**

### CODE REVIEW

File: projects/fusion-ui/components/button/v4/button.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<button [class]="buttonClass" [class.fu-loading]="loading" [disabled]="disabled || loading">
+    <fusion-icon *ngIf="iconName" class="fu-button-icon" [name]="iconName"></fusion-icon>
```
Comment (amitmo50) [inline projects/fusion-ui/components/button/v4/button.component.html:2]: What happens if iconName and startIconName is pass to the component?

File: projects/fusion-ui/components/button/v4/button.component.scss [hunk 2]
```diff
@@ -0,0 +1,232 @@
+@import '../../../src/style/scss/v4/colors';
+@import '../../../src/style/scss/v4/spacings';
+@import '../../../src/style/scss/v4/border-radius';
+@import '../../../src/style/scss/v4/vars/fonts';
+
+
+@mixin setButtonColor($color, $bg-color, $bg-color-hover, $outline-color, $outline-bg-color-hover){
```
Comment (amitmo50) [inline projects/fusion-ui/components/button/v4/button.component.scss:7]: We don't want to put it at mixin style lib?

File: projects/fusion-ui/components/loader/v4/loader.component.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    styleUrls: ['./loader.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LoaderComponent {
+    @Input() set strokeWidth(value: number) {
+        this._strokeWidth = value;
+    }
+
+    get strokeWidth(): number {
+        return this._strokeWidth ?? 2;
```
Comment (amitmo50) [inline projects/fusion-ui/components/loader/v4/loader.component.ts:18]: We don't want to put hard-coded values here no? lets put it on a constant file

**Review by amitmo50 — CHANGES_REQUESTED** [review]
Please see my comments

File: projects/fusion-ui/components/button/v4/button.component.html (hunk 1 above)
Comment (AndyKIron) [reply projects/fusion-ui/components/button/v4/button.component.html:2]: For this component it not in inputs.

File: projects/fusion-ui/components/button/v4/button.component.scss (hunk 2 above)
Comment (AndyKIron) [reply projects/fusion-ui/components/button/v4/button.component.scss:7]: This mixin relevant for the button component only.

File: projects/fusion-ui/components/loader/v4/loader.component.ts (hunk 3 above)
Comment (AndyKIron) [reply projects/fusion-ui/components/loader/v4/loader.component.ts:18]: Done.

**Review by amitmo50 — APPROVED** [review]

**Pull request merged**